apollo.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  client: {
+    service: {
+      name: 'client',
+      localSchemaFile: './schema.graphql',
+    },
+    tagName: 'omitGqlTagsTheyAreUnderGraphqlCodeGeneratorControl',
```

Comment:
Q1) Which plugin for VSCode do you use for `.graphql` files? It will be nice if it supports `ctrl+click` when clicking on fragment name and opens file with the fragment data.

`Apollo GraphQL` vscode plugin works quite well. 
It has two problems:
- need to relax via `tagName: "blaBlaBla"` option to not  parse gql`` tags in generated files (related issue: https://github.com/apollographql/apollo-tooling/issues/1715)
- does not provide autocompletion for available fragment names (but it's not so big problem)

Reply:
I'm just using the basic GraphQL plugin, it doesn't support jump to fragment :( 
A few months ago we tried to improve some of the existing plugins and add this functionality, but we didn't had the time to complete those... maybe we can collaborate and do that together :D